src/data/Restaurants.ts

```diff
@@ -0,0 +1,23 @@
+import { Cards } from "../models/Types";
+import { claroRestau,FourStarsIcon,Lumina,TigerLilly ,ThreeStarsIcon } from "../assets/homePhotos";
+const RestaurantsData: Cards = {
+  cards: [
+    {
+      title: "Claro",
+      image: claroRestau,
+      description: "Ran Shmueli",
+      rating:FourStarsIcon
```

Comment:
Consider making the stars (referring to the rating) more efficiently using what we discussed - understanding the structure of the SVG (such as path and fill) for rendering.